packages/core/lib/generators/assets_generator.dart

```diff
@@ -306,27 +307,30 @@ String _flatStyleDefinition(
       )
       .whereType<_Statement>()
       .toList();
-  return _flatStyleAssetsClassDefinition(statements);
+  final className = config.flutterGen.assets.outputs?.className;
+  return _flatStyleAssetsClassDefinition(className, statements);
 }
 
-String _flatStyleAssetsClassDefinition(List<_Statement> statements) {
+String _flatStyleAssetsClassDefinition(
+    String? className, List<_Statement> statements) {
```

Comment:
Please follow the current code convention. Please refer https://github.com/FlutterGen/flutter_gen/blob/main/packages/core/lib/generators/assets_generator.dart#L289